#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_change_Makefile.dpatch by Joao Eriberto Mota Filho <eriberto@eriberto.pro.br>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Make Makefile compliant with FHS.

@DPATCH@
diff -urNad pacman4console-1.2~/Makefile pacman4console-1.2/Makefile
--- pacman4console-1.2~/Makefile	2007-07-09 01:05:30.000000000 -0300
+++ pacman4console-1.2/Makefile	2007-07-09 01:11:33.000000000 -0300
@@ -2,16 +2,15 @@
 	gcc -lncurses pacman.c     -o pacman
 	gcc -lncurses pacmanedit.c -o pacmanedit
 
-install:	all
-	cp pacman /usr/local/bin
-	cp pacmanedit /usr/local/bin
-	mkdir -p /usr/local/share/pacman
-	cp -fR Levels/ /usr/local/share/pacman/
-	chown root:games /usr/local/bin/pacman
-	chown root:games /usr/local/share/pacman -R
-	chmod 750 /usr/local/bin/pacman
-	chmod 750 /usr/local/bin/pacmanedit
-	chmod 750 /usr/local/share/pacman/ -R
+install:
+	cp pacman $(DESTDIR)/usr/games/pacman4console
+	cp pacmanedit $(DESTDIR)/usr/bin/pacman4consoleedit
+	mkdir -p $(DESTDIR)/usr/share/pacman4console
+	cp -fR Levels/ $(DESTDIR)/usr/share/pacman4console/
+	chown root:games $(DESTDIR)/usr/games/pacman4console
+	chown root:games $(DESTDIR)/usr/share/pacman4console -R
+	chmod 755 $(DESTDIR)/usr/games/pacman4console
+	chmod 755 $(DESTDIR)/usr/bin/pacman4consoleedit
 
 uninstall:
 	rm -f /usr/local/bin/pacman
